### ISSUE TITLE
`BuyLiquidityBot` tries to buy on auctions which hasn't started yet

### ISSUE THREAD

**Issue by reverendus** [gnosis/dx-services#10]
The previous `MKR-DAI` auction finished around 25% of market price (I mean a real market price like on HitBTC) i.e. `146.88` as there were no buyers. So the next auction starts at 2x that (`293.77`). But as the real market price is round `450.00`, the `BuyLiquidityBot` instantly tries to buy. The problem is that it tries to do it even before the auction starts, causing exceptions in logs all the time:

```2018-08-29T18:22:05.554Z INFO-dx-service:services:LiquidityService-MKR-DAI We need to ensure that 100 % of the buy volume is bought. Market Price: 453.144, Price: 293.7732199330652, Relation: 64.82 %
2018-08-29T18:22:05.555Z INFO-dx-service:services:LiquidityService-MKR-DAI The auction has 0 % of the buy volume bought. So we need to buy an extra 100 %
2018-08-29T18:22:06.649Z INFO-dx-service:services:LiquidityService-MKR-DAI Posting a buy order for 265725.80930639565 DAI ($467715.50753646065)
2018-08-29T18:22:06.934Z ERROR-dx-service:bots:BuyLiquidityBot-MKR-DAI There was an error buy ensuring liquidity with the account 0xc32cac63823b556e6ebf61bb74149f08bf1aab34: AssertionError [ERR_ASSERTION]: The auction hasn't started yet
{ AssertionError [ERR_ASSERTION]: The auction hasn't started yet
    at AuctionRepoImpl.postBuyOrder (/usr/src/app/src/repositories/AuctionRepo/AuctionRepoImpl.js:867:5)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  generatedMessage: false,
  name: 'AssertionError [ERR_ASSERTION]',
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '==' }
2018-08-29T18:22:06.934Z ERROR-dx-service:error Uncought promise rejection: AssertionError [ERR_ASSERTION]: The auction hasn't started yet
{ AssertionError [ERR_ASSERTION]: The auction hasn't started yet
    at AuctionRepoImpl.postBuyOrder (/usr/src/app/src/repositories/AuctionRepo/AuctionRepoImpl.js:867:5)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  generatedMessage: false,
  name: 'AssertionError [ERR_ASSERTION]',
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,```

**Comment by reverendus** [comment]
Few minutes later, after the auction has already started, the buy was successful:

```2018-08-29T18:28:05.616Z INFO-dx-service:services:LiquidityService-MKR-DAI We need to ensure that 100 % of the buy volume is bought. Market Price: 453.144, Price: 293.46741852439095, Relation: 64.76 %
2018-08-29T18:28:05.616Z INFO-dx-service:services:LiquidityService-MKR-DAI The auction has 0 % of the buy volume bought. So we need to buy an extra 100 %
2018-08-29T18:28:06.690Z INFO-dx-service:services:LiquidityService-MKR-DAI Posting a buy order for 265449.20367561176 DAI ($467228.641983855)
2018-08-29T18:28:08.700Z INFO-dx-service:services:LiquidityService-MKR-DAI Posted a buy order. Transaction: undefined
2018-08-29T18:28:08.700Z INFO-dx-service:bots:BuyLiquidityBot-MKR-DAI I've bought 265449.20367561175676371658 DAI (467228.641983855 USD) in auction 2 to ensure BUY liquidity```

**Comment by anxolin** [comment]
Thanks for reporting! 

This was tried many times, and the buy bot don't do anything if the auction is in a waiting period.

I think this was a bug introduced recently. Please verify if `0.6.8` solves the issue. Otherwise Dani will take a look on what is happening.

**Comment by reverendus** [comment]
I don't think I will be able to verify it that easily as the auction price has already stabilized around the market price, so the situation described above just won't happen for me again :/ I could just stop the bots for some time and let the price fall again due to the lack of other participants, maybe I'll try to do it at some point.

**Comment by dasanra** [comment]
We have added a check to this state to show a more convenient message. Will be available from v2.0.3:

https://github.com/gnosis/dx-services/pull/50
